torchbiggraph/train.py

```diff
@@ -747,6 +747,18 @@ def log_status(msg, always=False):
         sync.barrier()
         dlog("All parts of the checkpoint have been written")
 
+        if config.checkpoint_interval is not None:
+            # If it is the right interval for saving, we copy all current
+            # checkpoints into it's own separate folder
```

Comment:
Nit: "its", not "it's".

Also, it's not "all current checkpoints" but "the entire current checkpoint".